### ISSUE TITLE
Is there anyway we can also download the video too?

### ISSUE THREAD

**Issue by HatimDa** [Yetangitu/Spodcast#28]
Hi Guys, thanks for this awesome repo, I managed to download the audio, is there a way to also include the video?

**Comment by Yetangitu** [comment]
Not really, I looked in to this and was stopped by a DRM-wall. I stopped looking as my goal for Spodcast was to enable RSS subscription to netcasts/podcasts, not video. It is probably possible by using a headless browser to get the video and either screen-scrape it or try to get at the decrypted stream in some other way but I have not (yet) put any effort into getting something like that to work.

**Comment by HatimDa** [comment]
Yeah, It's impossible to get the video, they protect it with DRM, tried every GitHub repo out there, but I think it's not possible to download it in one go because they sent only fractions of the podcast per request.

**Comment by Exabits1** [comment]
Not sure if you still need it, but someone just released a guide on how to get podcast videos. I'm downloading some videos now in case Spotify patches this method.

https://www.reddit.com/r/Piracy/comments/zzvc0t/guide_how_to_download_spotify_podcast_videos/
